pages/article/[articleLink].js

```diff
@@ -125,7 +130,22 @@ const Article = () => {
             )}
           </aside>
         </div>
-        <section className="mt-20 mb-10 border-t-2 border-accent-200">
+        {user && (
+          <section>
+            <h2 className="mb-5 mt-5 font-yrsa text-lg font-medium uppercase tracking-widest shadow-neutral-900">
+              Autor
+            </h2>
+            <Link href={`/user/${author}`}>
+              <a className="link">
+                {user?.name && user?.surname
+                  ? `${user.name} ${user.surname}`
+                  : user.username}
```

Comment:
This is absolutely correct code. Lemme show you one other approach how to "join" multiple strings together, always keeping only the existing ones, without "double spaces" or any random spaces

`[user?.name, user?.surname].filter(Boolean).join(' ')` It does 2 passes, so if the array will be 50 items, it might have impact on the perf. But since it is 2 items long. Whatever